services/app/src/components/views/quest/cardtemplates/decision/Actions.tsx

```diff
@@ -1,19 +1,69 @@
 import Redux from 'redux';
-import * as seedrandom from 'seedrandom';
+import {extractSkillCheck, Outcome, SkillCheck} from 'shared/schema/templates/Decision';
 import {QuestNodeAction, remoteify} from '../../../../../actions/ActionTypes';
-import {audioSet} from '../../../../../actions/Audio';
+import {toCard} from '../../../../../actions/Card';
+import {event} from '../../../../../actions/Quest';
 import {PLAYER_TIME_MULT} from '../../../../../Constants';
 import {AppStateWithHistory, MultiplayerState, SettingsType} from '../../../../../reducers/StateTypes';
+import {numLocalAndMultiplayerAdventurers, numLocalAndMultiplayerPlayers} from '../MultiplayerPlayerCount';
 import {ParserNode} from '../TemplateTypes';
-import SCENARIOS from './Scenarios';
-import {DecisionState, DecisionType, DIFFICULTIES, EMPTY_DECISION, EMPTY_SCENARIO, OutcomeType, PERSONA_TYPES, ScenarioType, SKILL_TYPES} from './Types';
+import {DecisionState, LeveledSkillCheck, RETRY_THRESHOLD_MAP, SUCCESS_THRESHOLD_MAP} from './Types';
 
-const NUM_SKILL_CHECK_CHOICES = 3;
+interface InitDecisionArgs {
+  node: ParserNode;
+  rp?: MultiplayerState;
+}
+export const initDecision = remoteify(function initDecision(a: InitDecisionArgs, dispatch: Redux.Dispatch<any>,  getState: () => AppStateWithHistory) {
+  if (!a.rp) {
```

Comment:
Yeah, there may be a couple "background" values that we could just expose via remoteify/argument inheritance so that they're commonly available to actions.